neurobooth_terra/redcap.py

```diff
@@ -344,7 +344,7 @@ def dataframe_to_tuple(df, df_columns, fixed_columns=None,
     for indicator_column in indicator_columns:
         mapping = dict()  # {race___1: 1, race___2: 2}
         for col in df.columns:
-            if col.startswith(indicator_column):
+            if col.startswith(indicator_column+'___'):
```

Comment:
It's not clear to me why you don't just change startswith to equals to avoid name collisions.

Reply:
Indicator columns get split into multiple columns of the form colName___<value number> for example race___1, race___2 and so on. The triple underscore is a delimiter that has been set in redcap iirc. The number following the colName is dependent on how many checkbox options are available. The actual meaning of what the numbers stand for comes from the redcap metadata_dictionary. Hence the use of startswith when written originally. Because the delimiter is fixed to triple underscore, a startswith(colName+delimiter) works the same functionally, but allows for other columns to exist with the form colName<_somethingElse>